release/llm_tests/serve/run_llm_serve_release_tests.py

```diff
@@ -42,9 +54,22 @@ def main(
     applications = get_applications(serve_config_file)
     compute_config = get_current_compute_config_name()
     env_vars = get_hf_token_env_var() if not skip_hf_token else {}
+    env_vars["VLLM_USE_V1"] = VLLM_USE_V1
+    llm_config = get_llm_config(serve_config_file)
 
+    if run_perf_profiler:
```

Comment:
Can we decouple running vllm perf profiling from the rest of other things happening here by using a different variable? maybe call it sth like `run_vllm_profiler`

We should also add docstrings/help descriptions for these variables (they are becoming several of them ).